security/azure/storage_account_https_enabled/README.md

```diff
@@ -1,5 +1,9 @@
 # Azure HTTPs for Storage Accounts
 
+## Deprecated
+
+This policy is no longer being updated. The [Azure Storage Accounts Without Secure Transfer](https://github.com/flexera-public/policy_templates/tree/master/security/azure/secure_transfer_required/) policy now includes this functionality.
+
 ## What it does
 
 This Policy checks for Azure Storage Accounts with HTTPs not enforced.
```

Comment:
Can't select it as it is not related to this change but - on Line 43 it should read `This Policy Template does not incur any cloud costs.`

Reply:
(Not really important as deprecated anyway)